Main/ClientApp/src/Components/Dialogs/ShareCollectionDialog.tsx

```diff
@@ -12,9 +23,29 @@ type ShareCollectionDialogProps = {
   collectionId: number;
 };
 
+type User = {
+  id: string;
+  name: string;
+  email: string;
+};
+
+enum UserRights {
```

Comment:
I would place it in Main/ClientApp/src/Model/User.ts, becouse it is a globally true